src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -203,9 +205,10 @@ else if (paymentGroup.getDisbursementType().getCode().equals(PdpConstants.Disbur
                     glPendingTransaction.setFinancialObjectCode(paymentAccountDetail.getFinObjectCode());
                     glPendingTransaction.setFinancialSubObjectCode(paymentAccountDetail.getFinSubObjectCode());
                 }
+
+                glPendingTransaction.setProjectCd(paymentAccountDetail.getProjectCode());
```

Comment:
so we are only setting the project code if we are NOT in a stale situation.  are we sure we always want project code to be empty in the stale check situation?

Reply:
Yes I have confirmed with the functionals that this is indeed the case.

Reply:
Thanks for confirming